recipes/opencolorio/all/conanfile.py

```diff
@@ -11,7 +11,8 @@ class OpenEXRConan(ConanFile):
     options = {"shared": [True, False], "fPIC": [True, False]}
     default_options = {"shared": False, "fPIC": True}
     generators = "cmake", "cmake_find_package"
-    exports_sources = ["patches/*"]
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    short_paths = True
```

Comment:
@p-podsiadly Is this for fixing a build error on MSVC?
@uilianries @danimtb Poking you guys again.

Reply:
Probably that same problem related to windows path in CCI, we need an internal fix for it.

Reply:
@p-podsiadly I think you can remove the `short_paths` now.